<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTPDlg</class>
 <widget class="QDialog" name="FTPDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="sourcePath_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>e:/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>源目录:</string>
   </property>
  </widget>
  <widget class="SourcefileWidget" name="sourcefileWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>731</width>
     <height>651</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="FTPPath_tableWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>160</y>
     <width>591</width>
     <height>221</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>IP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>端口</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择要导出的FTP路径：</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="FTP_SERVER_DIR">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>440</y>
     <width>591</width>
     <height>311</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>244</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>85</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>244</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>85</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>159</red>
         <green>158</green>
         <blue>158</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>159</red>
         <green>158</green>
         <blue>158</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>244</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>159</red>
         <green>158</green>
         <blue>158</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>180</number>
   </attribute>
   <attribute name="headerMinimumSectionSize">
    <number>56</number>
   </attribute>
   <column>
    <property name="text">
     <string>文件名 </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文件大小</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>修改时间</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>400</y>
     <width>591</width>
     <height>34</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="FTPSerVer_Label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>  Ftp服务器路径：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="DirEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ParentDirBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>120</y>
     <width>423</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="OutputFTPBtn">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>导出</string>
      </property>
      <property name="icon">
       <iconset resource="fastvideo.qrc">
        <normaloff>:/images/copyout.png</normaloff>:/images/copyout.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="AddFTPPathBtn">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>FTP路径</string>
      </property>
      <property name="icon">
       <iconset resource="fastvideo.qrc">
        <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="ModifyFTPPathBtn">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
      <property name="icon">
       <iconset resource="fastvideo.qrc">
        <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="DelFTPPathBtn">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
      <property name="icon">
       <iconset resource="fastvideo.qrc">
        <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SourcefileWidget</class>
   <extends>QWidget</extends>
   <header>sourcefilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="fastvideo.qrc"/>
 </resources>
 <connections/>
</ui>
